DataFormats/Math/interface/EigenComputations.h

```diff
@@ -0,0 +1,440 @@
+#ifndef DataFormats_Math_interface_EigenComputations_h
+#define DataFormats_Math_interface_EigenComputations_h
+
+#include <cmath>
+#include <limits>
+#include <type_traits>
+
+#include <Eigen/Dense>
+
+namespace calo {
+  namespace multifit {
```

Comment:
this namespace name seems to specific for the header

Reply:
Possibly.
Is that a problem ?

Reply:
It seems odd to put generic linear algebra functions in `calo::multifit`. From the top of my head, why not something like `reco::eigen`? Or otherwise, if these are really calorimeter-specific functions with no use elsewhere, then this header should be renamed/moved to denote that.
Let me know if this will be addressed.

Reply:
moved to a calorimeter-specific package